Validation: Added Missing Descriptions (#243)
MicrosoftDocs/bingmaps-docs: BingMaps/spatial-data-services/query-api/query-api-sample-code-csharp.md

---
title: "Query API Sample Code (C#) | Microsoft Docs"

> description: "Provides C# sample code for the Query API. This sample code provides an example of how to query a data source using the Query API."

Reviewer: ```suggestion
description: "The Query API Sample Code, written in C#, provides examples of how to query a data source using the Query API."
```
